app/models/etsource/inputs.rb

```diff
@@ -1,20 +1,23 @@
 module Etsource
-
   class Inputs
-    def initialize(etsource = Etsource::Base.instance)
-      @etsource = etsource
+    INPUTS = {
```

Comment:
Sorry for the continued nitpicking  (😞), but it feels a bit "magic" to initialize this with an Atlas class and then have an ETEngine class automatically selected to receive the data. I think it would be clearer if it were explicit:

* Provide both the Atlas and ETEngine classes
  
  ```ruby
  Etsource::Inputs.new(etsource, Atlas::Input, Input).import
  Etsource::Inputs.new(etsource, Atlas::InitializerInput, InitializerInput).import
  ```

* **Or** provide neither, and use a symbol key to identify which you are importing (still somewhat magic, but at least consistent in the params):

  ```ruby
  Etsource::Inputs.new(etsource, :input).import
  Etsource::Inputs.new(etsource, :initializer_input).import
  ```